tests/pfcwd/files/pfcwd_helper.py

```diff
@@ -364,3 +366,46 @@ def fetch_vendor_specific_diagnosis_re(duthost):
         return ""
 
     return VENDOR_SPEC_ADDITIONAL_INFO_RE.get(duthost.facts["asic_type"], "")
+
+def is_pfcwd_port_stormed( dut, storm_port ):
+   ''' storm_queue: Ethernet<num>:<queue_num>'''
+   stats = pfcwd_stats( dut )
+   for queue, status in stats.items():
+      if queue.startswith( storm_port ) and status == 'stormed':
+         return True
+   return False
+
+def pfcwd_stats( dut ):
+   result = dut.shell( "show pfcwd stats", module_ignore_errors=True, verbose=False )
+   if result['rc'] != 0:
```

Comment:
The result could be None, result['rc'] would raise the exception.

Reply:
Thanks, fixed